config/install/system.file.yml

```diff
@@ -0,0 +1,7 @@
+allow_insecure_uploads: false
+default_scheme: public
+path:
+  temporary: ''
```

Comment:
We will have to remember to set this for the LB and webheads so that it doesn't just default.

Reply:
https://stanfordits.atlassian.net/browse/D8CORE-335